skfp/distances/harris_lahey.py

```diff
@@ -280,16 +280,16 @@ def _bulk_harris_lahey_binary_similarity_single(
 ) -> np.ndarray:
     m = X.shape[0]
     sims = np.empty((m, m))
+    X_neg = 1 - X
```

Comment:
You are doubling memory usage here. I would prefer to avoid this

Reply:
Same in other places - this is already O(n^2) in memory due to `np.empty((m, m))`, and adding another O(n) here is a bad idea